plugins/inputs/huawei_routers_telemetry/huawei_routers_telemetry.go

```diff
@@ -0,0 +1,219 @@
+package huawei_routers_telemetry
+
+import (
+	"fmt"
+	"io"
+	"net"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/DamRCorba/huawei_telemetry_sensors"
+	"github.com/DamRCorba/huawei_telemetry_sensors/sensors/huawei-telemetry"
```

Comment:
Oh wow this was now costing me really a lot of time just to find out that the package you are importing here is internally names `package huawei_sensorPath`. This is totally unexpected! First you are thinking _"Where the heck is this used?"_ only to wonder _"Where does this 'huawei_sensorPath' thing come from?"_ in the code.

Please make this consistent by either renaming that imported package or move the repo.